### ISSUE TITLE
BUG: filesystem test noise due to quotas

### ISSUE THREAD

**Issue by pcmoore** [SELinuxProject/selinux-testsuite#83]
When running the test suite on recent Fedora Rawhide systems I see the following messages:
```
Running as user root with context unconfined_u:unconfined_r:unconfined_t

domain_trans/test ........... ok   
 ...
filesystem/ext4/test ........ 3/83 quotacheck: Your kernel probably supports 
  ext4 quota feature but you are using external quota files. Please switch 
  your filesystem to use ext4 quota feature as external quota files on ext4 
  are deprecated. You can enable the feature by unmounting the file system
  and running 'tune2fs -O quota <device>'.
 ...
```

**Comment by WOnder93** [comment]
I'm already working on this.

**Comment by pcmoore** [comment]
Thanks!

**Comment by WOnder93** [comment]
Patch posted: https://lore.kernel.org/selinux/20230110135513.56005-1-omosnace@redhat.com/

**Comment by WOnder93** [comment]
Fixed with bbab270f66c3fc33b4fdc7cec8beb0003afbb4ee.
